src/security/store.js

```diff
@@ -73,13 +75,10 @@ const actions = {
     context.commit(LOGOUT);
     localStorage.removeItem("token");
     localStorage.removeItem("user");
+    localStorage.removeItem("token_expire");
   },
   getUser: (context, token) => {
-    const access_token = token
-      ? token
-      : localStorage.getItem("token")
-        ? localStorage.getItem("token")
-        : false;
+    const access_token = token || localStorage.getItem("token") || false;
     if (access_token) {
       return new Promise((resolve, reject) => {
         UserProvider.provide(access_token)
```

Comment:
I don't know much about vue approach but it sounds like something is wrong here with Promises. I feel like you unnecessary nesting Promises. We can discuss it later if you want